src/main/java/nl/novi/techiteasycontroller/service/TelevisionService.java

```diff
@@ -0,0 +1,258 @@
+
+        return television;
+
+    }
+
+    public TelevisionOutputDto transferToDto(Television television) {
+        TelevisionOutputDto televisionOutputDto = new TelevisionOutputDto();
+
+        televisionOutputDto.id = television.getId();
+        televisionOutputDto.type = television.getType();
+        televisionOutputDto.name = television.getName();
+        televisionOutputDto.brand = television.getBrand();
+        televisionOutputDto.price = television.getPrice();
+        televisionOutputDto.availableSize = television.getAvailableSize();
+        televisionOutputDto.refreshRate = television.getRefreshRate();
+        televisionOutputDto.screenType = television.getScreenType();
+        televisionOutputDto.screenQuality = television.getScreenQuality();
+        televisionOutputDto.smartTv = television.getSmartTv();
+        televisionOutputDto.wifi = television.getWifi();
+        televisionOutputDto.voiceControl = television.getVoiceControl();
+        televisionOutputDto.hdr = television.getHdr();
+        televisionOutputDto.bluetooth = television.getBluetooth();
+        televisionOutputDto.ambiLight = television.getAmbiLight();
+        televisionOutputDto.originalStock = television.getOriginalStock();
+        televisionOutputDto.sold = television.getSold();
+
+        return televisionOutputDto;
+    }
+
+    public void assignRemoteControllerToTelevision(Long id, Long remoteId) {
+    var optionalTelevision = this.televisionRepository.findById(id);
+    var optionalRemoteController = this.remoteControllerRepository.findById(remoteId);
+
+    if(optionalTelevision.isPresent() && optionalRemoteController.isPresent()){
+        var television = optionalTelevision.get();
+        var remoteController = optionalRemoteController.get();
+
+        television.setRemoteController(remoteController);
+        this.televisionRepository.save(television);
+    }
+    else{
+        throw new RecordNotFoundException();
+    }
+    }
+
+    public void assignCIModuleToTelevision(Long id, Long ciModuleId){
+        var optionalTelevision = this.televisionRepository.findById(id);
+        var optionalRemoteController = this.remoteControllerRepository.findById(ciModuleId);
+
+        if(optionalTelevision.isPresent() && optionalRemoteController.isPresent()){
+            var television = optionalTelevision.get();
+            var remoteController = optionalRemoteController.get();
+
+            television.setRemoteController(remoteController);
+            this.televisionRepository.save(television);
+        }
+        else{
+            throw new RecordNotFoundException();
+        }
+    }
+
+    public void assignWallBracketToTelevision(Long id, WallBracket wallBracket){
+        var OptionalTelevision = this.televisionRepository.findById(id);
+
+        if(OptionalTelevision.isPresent()){
+            var television = OptionalTelevision.get();
+
+            television.getWallBrackets().add(wallBracket);
```

Comment:
Het is de bedoel dat je Wallbracket wel een "persisted state" heeft als je het aan de Television toevoegd, oftewel, het moet een gesavete wallbracket zijn. 

Dit kun je op twee manieren regelen:

- OF je zorgt dat je de Wallbracket uit de parameter hier eerst saved in de wallbracket repository en dan pas aan de Television toevoegd

- OF je zorgt dat je geen Wallbracket, maar een Long (een wallbracketid dus) binnen krijgt. Dan kun je de Wallbracket uit de WallbracketRepository halen met findById. Als die gevonden wordt, kun je die toevoegen aan de Television, anders gooi je een Exception op.